ui/src/app/views/workspace/component.spec.ts

```diff
@@ -125,8 +124,8 @@ describe('WorkspaceComponent', () => {
     registerApiClient(WorkspacesApi, new WorkspacesApiStub());
   }));
 
-  it('displays research purpose', fakeAsync(() => {
-    expect(fixture.debugElement.queryAll(
-      By.css('.research-purpose-item')).length).toEqual(2);
+  // TODO [RW-2594] add new tests
```

Comment:
If you aren't adding tests here, would it be worth throwing a quick comment with what tests you think should be added? Alternatively, is there a reason _not_ to just add the tests?

Reply:
reason not to add the tests is that the ws about page is about to be refactored in RW-2594.  tests here should probably test that it displays the correct answers to rp questions as per the new UI

Reply:
Got it. Can you add that as a comment?